ozaria/site/components/play/PageIntroLevel/index.vue

```diff
@@ -0,0 +1,134 @@
+<script>
+import api from 'core/api'
```

Comment:
Don't forget to lint!  Script blocks have an indentation of 2.

Reply:
I tried using eslint in vscode but it didnt give me any errors for vue files. Can you point me in the right direction for the tool to use for linting here?

Reply:
+1 - Also can't get eslint to work for vue (in the editor)

Reply:
@AndrewJakubowicz @shubhi1092 I don't use VSCode so I'm not sure.  I did a quick search and came across this article that references some additional configuration.  https://alligator.io/vuejs/eslint-vue-vetur/

Reply:
Yep, this worked! Thanks @jmif, I will make sure to lint the vue files as a separate PR (don't want to mix those changes with this PR)

Reply:
👍